README.md

```diff
@@ -115,7 +115,9 @@ And then run your builds as above appending `--volumes-from builder_cache` to th
 
 ## Adding repository credentials
 
-If your tests depend on private images, you can pass their credentials either by mounting your local `.docker` folder inside the container appending `-v $HOME/.docker:/.docker:ro`, or by providing the contents of the `.docker/config.json` file via an environment variable called `$DOCKERCFG`: `-e DOCKERCFG=$(cat $HOME/.docker/config.json)`
+If your tests depend on private images, you can pass their credentials either by mounting your local `.docker` folder inside the container appending `-v $HOME/.docker:/.docker:ro`, or by providing the contents of the docker configuration file.
+* For order version, put content of `$HOME/.dockercfg` via an environment variable called `$DOCKERCFG`. eg `-e DOCKERCFG="$(cat $HOME/.dockercfg)"`
```

Comment:
Hi @fermayo , earlier it seems that you want backward compatibility. "1. Keep $DOCKERCFG being echoed to ~/.dockercfg for backwards compatibility. We might show a warning if this is given." So are you saying we are OK with remove $DOCKERCFG? If so, I'd happily do that for you.